packages/automated_actions/pyproject.toml

```
@@ -9,14 +9,21 @@ requires-python = "~= 3.12.0"
 dependencies = [
     "celery[sqs]==5.4.0",
     "fastapi==0.115.12",
+    "gql==3.5.0",
     "httpx==0.28.1",
+    "hvac>=2.3.0",
     "itsdangerous==2.2.0",
+    "kubernetes==32.0.1",
+    "openshift==0.13.2",
     "prometheus-fastapi-instrumentator==7.1.0",
     "pydantic==2.10.6",
     "pydantic-settings==2.8.1",
     "pyjwt==2.10.1",
     "pynamodb==6.0.2",
+    "requests-toolbelt==1.0.0",
+    "types-hvac>=2.3.0.20240621",
```

Please pin the version
```suggestion
    "types-hvac==2.3.0.20240621",
```

---

https://github.com/app-sre/automated-actions/pull/51